SPECS/dbus/dbus.spec

```diff
@@ -66,32 +68,34 @@ install -vdm755 %{buildroot}%{_libdir}
 make %{?_smp_mflags} check
 
 %files
-%defattr(-,root,root)
```

Comment:
Why the removal of `defattr`?

Reply:
Good catch, I forgot to readd this following some testing.